### PR TITLE
AAP-3713: New installable RPM

### PR DESCRIPTION
**Pull request by lmalivert**
Jira: [AAP-3713](https://issues.redhat.com/browse/AAP-3713)

- Added rpm install instructions.
- Reordered _**Choosing and obtaining a Red Hat Ansible Automation Platform installer**_ section to go after the _**Attaching your Red Hat Ansible Automation Platform subscription**_ section.

### CODE REVIEW

**Review by Spredzy — APPROVED** [review]

File: downstream/modules/platform/proc-choosing-obtaining-installer.adoc [hunk 1]
```diff
@@ -15,7 +15,9 @@ A valid Red Hat customer account is required to access {PlatformName} installer
 
 .Installing with internet access
 
-Choose the {PlatformName} installer if your Red Hat Enterprise Linux environment is connected to the internet. Installing with internet access will retrieve the latest required repositories, packages, and dependencies.
+Choose the {PlatformName} (AAP) installer if your Red Hat Enterprise Linux environment is connected to the internet. Installing with internet access will retrieve the latest required repositories, packages, and dependencies. Choose one of the following ways to set up your AAP installer.
```
Comment (dcdacosta) [inline downstream/modules/platform/proc-choosing-obtaining-installer.adoc:18]: Suggest active voice for second sentence: Installing with internet access retrieves...

**Review by dcdacosta — APPROVED** [review]
One minor suggestion, otherwise, it LGTM.

File: downstream/modules/platform/proc-choosing-obtaining-installer.adoc (hunk 1 above)
Comment (lmalivert) [reply downstream/modules/platform/proc-choosing-obtaining-installer.adoc]: Thanks, @dcdacosta. Can you please provide a final review and then squash and merge?

**Comment by dcdacosta** [comment]
LGTM!

**Pull request merged**